main.js

```diff
@@ -0,0 +1,83 @@
+//querySelectors
+
+var addRecipeBtn = document.querySelector('.recipe-btn');
+var letsCookBtn = document.querySelector('.cook-btn');
+
+//eventListeners
+//buttons
+//does the below button even need functionality?
```

Comment:
good question, I think this is one of the extensions. So it will only need functionality if you choose to do that extension

Reply:
from the [comp](https://frontend.turing.edu/projects/module-1/dinner.html), Don’t worry about the “Add a Recipe” button and “Entire Meal” radio button for now.